openpdf/src/test/java/com/lowagie/text/pdf/TextExtractTest.java

```diff
@@ -59,6 +59,8 @@ void textCreateAndExtractTest2() throws IOException {
         // FileOutputStream test = new FileOutputStream("/tmp/output2.pdf");
         // pdfOutput.writeTo(test);
 
-        Assertions.assertEquals("ก ข น ํ้ า ต า ญูญูิ่ ก้กิ้", pdfTextExtractor.getTextFromPage(1));
+        // Ignore spaces in comparison
+        Assertions.assertEquals("ก ข น ํ้ า ต า ญูญูิ่ ก้กิ้".replaceAll(" ", ""),
+            pdfTextExtractor.getTextFromPage(1).replaceAll(" ", ""));
```

Comment:
I really have no idea what is being tested here, and how good is the test and the implementation. Just a comment.